plugins/discovery/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	download.Config         // bundle downloader configuration
 	Name            *string `json:"name"`   // name of the discovery bundle
 	Prefix          *string `json:"prefix"` // path prefix for downloader
+	QueryName       *string `json:"query"`  // path prefix for downloader
```

Comment:
I chatted with @tsandall about this a little bit this morning. The outcome of the discussion is that we think it would make more sense to change this to `decision` (so it would be `discovery.decision` in the config). Similar to what the `default_decision` and `default_authorization_decision` options look like. This includes supporting the `/` notation like they do for consistency https://www.openpolicyagent.org/docs/latest/configuration/#miscellaenous